docs/content/users/configuration/config.md

```diff
@@ -237,6 +237,22 @@ Port for project’s MailHog HTTP URL.
 | -- | -- | --
 | :octicons-file-directory-16: project | `8025` | Can be changed to avoid a port conflict.
 
+## `messages:disable_ticker`
+
+Whether to disable ticker messages.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-globe-16: global | `false` | Can be `true` or `false`; only `true` has any effect.
+
+## `messages:ticker_interval`
+
+How often ticker messages should be shown.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-globe-16: global | `20` | Interval in hours.
+
```

Comment:
Agree to combine the two, that's sound good.

The user shouldn't change this setting in the first place and that's why I don't have provided a config command flag.

And this change introduces sub structures in the yaml. So far we only had one level with all options. To have cleaner yaml files I've introduced this sub structures like we already know e.g. from docker-compose. This is what the ":" should outline.

Reply:
My understanding is that this YAML config is for any user, and DDEV doesn’t require any familiarity with Docker and Docker Compose for basic setup. So while I understand the `:` should be a familiar thing from Docker Compose, it’s still a new convention in this format. (Commands like `service` have sub-commands, so there’s sort of a precedent for that already.) That’s why I dropped `messages:` from my suggestion altogether, even though I like the idea of using that convention if it’s done more broadly. A one-off change seems more confusing.

If the user shouldn’t change this setting, wouldn’t we want to tell them that directly? Or _not_ document the setting?

I may be part of a rare and needlessly grumpy contingent, but I would immediately disable any messaging that adds noise to the experience without any clear value to me. (Which would probably be concise new feature announcements, or specific suggestions based on how I use DDEV.) This might be one of the first things I figure out how how to disable as a new user. Or it might motivate me to contribute to the project, with the perk of quieting CTAs exactly like this.) So if I’m to change this setting, all the more helpful that I could change it like most other values: as a command flag or YAML setting.

Do whatever you will (or not!) with this feedback, I’m just very sensitive about trying not to take the user’s attention for granted and this feature appears to introduce marketing messages into the daily DDEV experience.

Reply:
Btw most messages should be about the usage of DDEV, no marketing at all. Only one message to make people more sensitive to sponsor DDEV is planed so far.